ArduPlane/RC_Channel.cpp

```diff
@@ -32,6 +32,7 @@ void RC_Channel_Plane::init_aux_function(const RC_Channel::aux_func_t ch_option,
 {
     switch(ch_option) {
     // the following functions do not need to be initialised:
+    case MOTOR_ESTOP:
```

Comment:
I think this means that if the switch starts in high position then motors won't initialised stopped?
I think estop should be enabled if the switch starts high

Reply:
we should ensure the behaviour is the same as in Copter.  At some stage we should have this code common amongst vehicles.  Fewer places to make mistakes in a safty-critical part of the code.

It's not noted in here but @WickedShell pointed out on the devcall that the servo library already kinda-sorta knows which outputs have motors on them.

All our other vehicles could do with the concept of an estop.

Reply:
I will see what I can do